mathics/builtin/system.py

```diff
@@ -36,99 +36,40 @@
 sort_order = "mathics.builtin.global-system-information"
 
 
-class MaxLengthIntStringConversion(Predefined):
-    """
-    <url>:Python 3.11 Integer string conversion length limitation:
-    https://docs.python.org/3.11/library/stdtypes.html#int-max-str-digits</url>
+class Breakpoint(Builtin):
+    """<url>:Python breakpoint:https://docs.python.org/3/library/functions.html#breakpoint</url>
+
     <dl>
-      <dt>'$MaxLengthIntStringConversion'
-      <dd>A positive system integer that fixes the largest size of the string that \
-          can appear when converting an 'Integer' value into a 'String'. When the \
-          string value is too large, then the middle of the integer contains \
-          an indication of the number of digits elided inside << >>.
+      <dt>'Breakpoint[]'
+      <dd> Invoke a Python breakpoint.
 
-          If '$MaxLengthIntStringConversion' is set to 0, there is no \
-          bound. Aside from 0, 640 is the smallest value allowed.
+      This can be used for debugging the Mathics3 implementation, but \
+      if you are familiar with Python, it might assist in debugging a Mathics3 programs \
+      as well.
 
-          The initial value can be set via environment variable \
-          'DEFAULT_MAX_STR_DIGITS'. If that is not set, \
-          the default value is 7000.
+      By default, the Python debugger ('pdb') is loaded. For loading other debuggers, \
+      the change the environment variable 'PYTHONBREAKPOINT'.
     </dl>
 
-    Although Mathics3 can represent integers of arbitrary size, when it formats \
-    the value for display, there can be nonlinear behavior in printing the decimal string \
-    or converting it to a 'String'.
-
-    Python, in version 3.11 and up, puts a default limit on the size of \
-    the number of digits allows when converting a large integer into \
-    a string.
-
-    Show the default value of '$MaxLengthIntStringConversion':
-    >> $MaxLengthIntStringConversion
-     = ...
-
-    500! is a 1135-digit number:
-    >> 500! //ToString//StringLength
-     = ...
-
-    We first set '$MaxLengthIntStringConversion' to the smallest value allowed, \
-    so that we can see the truncation of digits in the middle:
-    >> $MaxLengthIntStringConversion = 640
-    ## Pyston 2.3.5 returns 0 while CPython returns 640
-    ## Therefore output testing below is generic.
-     = ...
 
-    Note that setting '$MaxLengthIntStringConversion' has an effect only on Python 3.11 and later;
-    Pyston 2.x however ignores this.
+    Mathics3 also provides "breakpoint" handler 'mathics.disabled_breakpoint' which \
+    notes that a 'Breakpoint[]' was encountered, but does not stop inside Python. \
```

Comment:
```suggestion
    notes that a breakpoint was encountered, but does not stop inside Python. \
```

The breakpoint could be, e.g., reached due to a `break` debugger command as well.

Reply:
A debugger `break` would not trigger this. Adding to the source code `breakpoint()` would though. I will revise.